_posts/2024-06-06-switch-to-bam.md

```diff
@@ -13,10 +14,13 @@ contributors:
 reviewers:
   - pmenzel
   - hexylena
+last_modified_at: 2024-06-07
 ---
 
 uBAM is a strictly better storage format for your sequencing data, and you should switch to it today.
 
+{% link assets/images/ubam-gather-round %}
```

Comment:
missing the file extension

Reply:
I fixed the extension and renamed the file